dali/operators/reader/loader/loader.h

```diff
@@ -228,18 +228,35 @@ class Loader {
     }
     return SizeImpl();
   }
+
   // Give the size of the data accessed externally from python
-  Index SizePadded() {
+  Index SizePadded(bool consider_padding = true) {
     if (!loading_flag_) {
```

Comment:
Nitpick maybe, but why is it still called "SizePadded" if padding can be ingored?

Reply:
Done